docker/fetch_tezos_sources.py

```diff
@@ -40,6 +37,5 @@
 
 os.chdir("opam-repository")
 subprocess.run(["git", "checkout", opam_repository_tag])
-subprocess.run(["rm", "-rf", ".git"])
-subprocess.run(["rm", "-r", "zcash-params"])
```

Comment:
Likewise, this was done to save space AFAIK, why is it now necessary to avoid removing these subfolders?